### PR TITLE
[KARAF-6652] fixed broken links

### PR DESCRIPTION
**Pull request by oklimberg**
additionally, using HTTPS for all links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 # Apache Karaf
 
-[Apache Karaf](http://karaf.apache.org) is a modern and polymorphic applications runtime.
+[Apache Karaf](https://karaf.apache.org) is a modern and polymorphic applications runtime.
```
Comment (jbonofre) [inline README.md]: FYI, http://karaf.apache.org is still valid.

**Comment by oklimberg** [comment]
No, just thought, changing them, so the 'not secure' warning in most of the browsers are gone and users gain trust.

**Comment by jbonofre** [comment]
Fair enough. Just wondering if you had issue with http.

**Comment by jbonofre** [comment]
No your change is fine. Just wanted to check if there were issues with http.

**Comment by oklimberg** [comment]
No problem

**Comment by jbonofre** [comment]
retest this please

**Pull request merged**